components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -104,10 +107,57 @@ function Main() {
     }
   }
 
+  let inputFocused: boolean = false
+  let viewPortWithoutKeyboard = 0
+  let keyboardSize = 0
+
+  window.addEventListener('resize', () => {
+    if (!context.isMobile || !inputFocused || window.visualViewport == null) {
```

Comment:
Additionally, I'm curious about whether we could use the solution mentioned here:
https://stackoverflow.com/a/75243097/3260044

And just use 100% (or 100vh)